mathml/relations/css-styling/lengths-3.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>MathML lengths</title>
+<link rel="help" href="http://www.mathml-association.org/MathMLinHTML5/S2.html#SS3.SSS1"/>
+<meta name="assert" content="Verify various cases of the MathML length syntax.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+  @font-face {
+    font-family: TestFont;
+    src: url("/fonts/math/xheight500.woff");
+  }
+  math {
+    font-family: TestFont;
+    font-size: 10px;
+  }
+</style>
+<script>
+  var epsilon = 1;
```

Comment:
Seems like at least most of these comparisons should be exact?

Picking a multiplier for named spaces that is a multiple of 18 could make their use exact.

I'm not clear about the intentions for the different fractions in non-negative and non-positive numbers.  These could be tested less precisely than those expected to be exact, but still more precisely than a 2 pixel accepted range.